twittermap/public/stylesheets/main.css

```diff
@@ -18,7 +18,7 @@ body {
   bottom: 0;
   left: 96%;
   width: 24%;
-  height: 700px;
+  height: 90%;
```

Comment:
I think it's better to be relative than absolute in general. But I don't know if there is any reason for choosing this particular `700px`. @hh1680651  you may want to change the resolution using develop mode to confirm that the new value `90%` won't cause much trouble?